src/home_robot/home_robot/motion/stretch.py

```diff
@@ -645,9 +656,11 @@ def manip_ik(
         By default move relative. easier that way.
         """
 
+        # TODO: the following does nothing as it is not an in-place change; the result needs to be stored
         if q0 is not None:
-            self._to_manip_format(q0)
-            default_q = q0
+            default_q = self._to_manip_format(
+                q0
+            )  # FIXME: this default_q will raise error when passed to PIN
```

Comment:
Add more context on what errors are being raised? Can also be put into an issue.